Apps/Encounter/test/unit/Services/EncounterServiceTests.cs

```diff
@@ -410,15 +421,17 @@ public async Task GetHospitalVisitsShouldReturnError()
 
         private static IConfigurationRoot GetIConfigurationRoot()
         {
-            Dictionary<string, string?> configuration = new()
+            Dictionary<string, string> configuration = new()
             {
                 { "PHSA:BaseUrl", ConfigBaseUrl },
                 { "PHSA:FetchSize", ConfigFetchSize },
-                { "PHSA: BackOffMilliseconds", ConfigBackOffMilliseconds },
+                { "PHSA:BackOffMilliseconds", ConfigBackOffMilliseconds },
             };
 
+            configuration["MspVisit:ExcludedFeeDescriptions:1"] = "PRIMARY CARE PANEL REPORT";
+
             return new ConfigurationBuilder()
-                .AddInMemoryCollection(configuration.ToList())
+                .AddInMemoryCollection(configuration!)
```

Comment:
```suggestion
                .AddInMemoryCollection(configuration)
```
Exclamation shouldn't be needed now that variable isn't nullable.

Reply:
It looks like it is required.

<img width="2525" alt="Screenshot 2023-10-16 at 2 25 25 PM" src="https://github.com/bcgov/healthgateway/assets/58790456/c3a6890f-7b7a-4b49-819b-de084bf8e0e7">

Reply:
Changed to Dictionary<string, string?>